src/rules/strictComparisonsRule.ts

```diff
@@ -276,6 +354,26 @@ function isEqualityOperator(node: ts.BinaryExpression): boolean {
     }
 }
 
+function getKindFromSyntaxKind(type: ts.SyntaxKind): TypeKind {
```

Comment:
Later on, in the only use of `getKindFromSyntaxKind`, the returned value is cast `as ts.SyntaxKind`. Maybe that's what should be returned here?